### ISSUE TITLE
SA1002 code fix bug

### ISSUE THREAD

**Issue by pdelvo** [DotNetAnalyzers/StyleCopAnalyzers#403]
For this code:
```csharp
            return contract == null || contract.UnderlyingType == typeof(object) || contract.ContractType == JsonContractType.Linq
                    || contract.UnderlyingType == typeof(IDynamicMetaObjectProvider)
;
```
The ; on the last line is correctly diagnosed as SA1002. It has a "Fix spacig" code fix which does not change the document.

**Comment by Maxwe11** [comment]
>The ; on the last line is correctly diagnosed as SA1002.

Why? [Rule](http://www.stylecop.com/docs/SA1002.html)
>a semicolon should never be preceded by any whitespace, unless it is the first character on the line.
